webpack/assets/javascripts/react_app/redux/index.js

```diff
@@ -2,20 +2,27 @@ import createLogger from 'redux-logger';
 import thunk from 'redux-thunk';
 import { applyMiddleware, createStore } from 'redux';
 
-import reducer from './reducers';
+import combineReducersAsync from './reducers';
 
 let middleware = [thunk];
+const asyncReducers = {};
 
 if (process.env.NODE_ENV !== 'production' && !global.__testing__) {
   middleware = [...middleware, createLogger()];
 }
 
-const _getStore = () => createStore(
-  reducer,
-  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__(),
-  applyMiddleware(...middleware),
-);
+export const generateStore = () =>
```

Comment:
I renamed this to `generateStore` since it's actually generating a new store each time. The main application store is exported as the default but test files use `generateStore` to generate a unique store for the test suite.